src/index.js

```diff
@@ -49,19 +49,22 @@ function parse(model, response, options = {}) {
 export default Bookshelf => {
   const Model = Bookshelf.Model.prototype;
   const client = Bookshelf.knex.client.config.client;
-  const parseOnFetch = client === 'sqlite' || client === 'sqlite3';
+  const handleOnSaving = client !== 'mysql';
```

Comment:
Disabling serialization on save for mysql breaks in my project, so I'm inclined to think that you should not do this.  It seems like it is necessary to allow the plugin to do its thing both on save and fetch for mysql.  Disabling on save when trying to save a model with an object in place of the expected JSON string results in the following error:  `Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1`.  So the json column / object still needs to be serialized/stringified when mysql is the client.